leasing/report/lease/reservations.py

```diff
@@ -63,6 +63,87 @@ def get_reservation_procedure(obj):
     return obj.reservation_procedure.name if obj.reservation_procedure else None
 
 
+def get_plan_unit_intended_uses(obj):
+    plan_units_intended_uses = []
+    for lease_area in obj.lease_areas.all():
+        if lease_area.archived_at:
+            continue
+        for plan_unit in lease_area.plan_units.all():
+            if plan_unit.plan_unit_intended_use:
+                plan_units_intended_uses.append(plan_unit.plan_unit_intended_use.name)
+
+    return ", ".join(plan_units_intended_uses)
+
+
+def get_plan_unit_detailed_plan_identifier(obj):
+    plan_units_detailed_plan_identifiers = []
+    for lease_area in obj.lease_areas.all():
+        if lease_area.archived_at:
+            continue
+        for plan_unit in lease_area.plan_units.all():
+            if plan_unit.detailed_plan_identifier:
+                plan_units_detailed_plan_identifiers.append(
+                    plan_unit.detailed_plan_identifier
+                )
+
+    return ", ".join(plan_units_detailed_plan_identifiers)
+
+
+def get_plan_unit_usage_distribution_main_build_permission(obj):
+    """
+    build permission of usage distribution with highest build permission
+    """
+    main_build_permission = 0
+    for lease_area in obj.lease_areas.all():
+        if lease_area.archived_at:
+            continue
+        for plan_unit in lease_area.plan_units.all():
+            for usage_distribution in plan_unit.usage_distributions.all():
+                if int(usage_distribution.build_permission) > main_build_permission:
```

Comment:
Will each usage_distribution always have `build_permission` that `int()` will accept?